src/services/token.service.js

```diff
@@ -0,0 +1,29 @@
+import { Token } from '../models/token.js';
+
+async function save(userId, newToken) {
+  const token = await Token.findOne({ where: { userId } });
```

Comment:
The query should use 'userId' if it matches the field name in the database. Ensure that the field name used here matches the actual database schema.